app/services/idv/analytics_events_enhancer.rb

```diff
@@ -183,8 +183,24 @@ def profile_history
     end
 
     def proofing_components
-      return if !user&.respond_to?(:proofing_component) || !user.proofing_component
-      ProofingComponentsLogging.new(user.proofing_component)
+      return if !user
+
+      idv_session = Idv::Session.new(
+        user_session: session&.dig('warden.user.user.session') || {},
+        current_user: user,
+        service_provider: sp,
+      )
+
+      proofing_components_hash = ProofingComponents.new(
+        user:,
+        idv_session:,
+      ).to_h
+
+      proofing_components_hash.empty? ? nil : proofing_components_hash
     end
   end
+
+  def user_session
```

Comment:
this method looks unused? also at the wrong indentation level (aka defined on `Idv` module instead of `AnalyticsEventsEnhancer`

Reply:
Oof, thank you. Addressed in d6a4cda